src/tlo/methods/consumables.py

```diff
@@ -200,6 +200,7 @@ def _determine_default_return_value(cons_availability, default_return_value):
     def _request_consumables(self,
                              facility_info: 'FacilityInfo',  # noqa: F821
                              item_codes: dict,
```

Comment:
to be explicit shall we rename this kwarg "essential_item_codes"

Reply:
done